test/tables.txt

```diff
@@ -1165,3 +1204,39 @@ Here's a link to [Freedom Planet 2][].
 <td align="center">r3c2</td>
 </tr></tbody></table>
 ````````````````````````````````
+
+### Invalid table (MM-55972/MM-59809)
```

Comment:
Is there any benefit in telling where it's invalid similar to this [comment](https://mattermost.atlassian.net/browse/MM-59809?focusedCommentId=168645)?

Reply:
Good idea. I'll add that